tests/bearlib/languages/documentation/DocBaseClassTest.py

```diff
@@ -254,5 +255,25 @@ def test_extract_documentation_PYTHON3_4(self):
         # Nothing is yielded as triple quote string literals are being
         # ignored.
         self.assertEqual(
-            list(extract_documentation(data, 'PYTHON3', 'default')),
+            list(DocBaseClass.extract(data, 'PYTHON3', 'default')),
             [])
+
+    def test_diff_generation(self):
+        # As diff requires filename
+        dataname = 'dummy_file.py'
+
+        data_old = ['\n', '""" documentation in single line  """\n']
+        for doc_comment in DocBaseClass.extract(
+                                data_old, 'PYTHON3', 'default'):
+            old_doc_comment = doc_comment
+
+        data_new = ['\n', '"""\n documentation in single line\n"""\n']
+        for doc_comment in DocBaseClass.extract(
+                                data_new, 'PYTHON3', 'default'):
+            new_doc_comment = doc_comment
+
+        (affected_code, diffs) = DocBaseClass.generate_diff(
```

Comment:
We dont need this now with the new changes I guess. I will directly compare the diff object.